WalletWasabi.Fluent.Generators/Generators/AutoInterfaceGenerator.cs

```diff
@@ -109,25 +109,25 @@ private void Execute(ClassDeclarationSyntax cls)
 					  .Select(x => $"using {x};")
 					  .ToList();
 
-		var source = @$"// <auto-generated />
-
-#nullable enable
-{string.Join("\r\n", namespaces)}
+		var source = $$"""
+		               // <auto-generated />
```

Comment:
Why here we have mixed tabs and spaces?

Reply:
IDE is evil, moved string start to a new line, seems to make it easier to indent

Reply:
Pushed fixes

Reply:
ok more fixes pushed